arisa.json

```diff
@@ -0,0 +1,122 @@
+
+    "modules": {
+      "attachment": {
+        "resolutions": [
+          "Awaiting Response",
+          "Cannot Reproduce",
+          "Done",
+          "Duplicate",
+          "Fixed",
+          "Incomplete",
+          "Invalid",
+          "Unresolved",
+          "Won't Fix",
+          "Works As Intended"
+        ],
+        "extensionBlacklist": ["jar", "exe", "com", "bat", "msi", "run", "lnk", "dmg"]
+      },
+
+      "piracy": {
+        "message": "*Thank you for your report!*\nHowever, this issue is {color:#FF5722}*Invalid*{color}.\n\nYou are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].\nWe will not provide support for pirated versions of the game, these versions are modified and may contain malware.\n\n\n*Quick Links*:\n\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]\n~{color:#888}-- I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}~",
+        "piracySignatures": [
+          "Minecraft Launcher null",
+          "Bootstrap 0",
+          "Launcher: 1.0.10  (bootstrap 4)",
+          "Launcher: 1.0.10  (bootstrap 5)",
+          "Launcher 3.0.0",
+          "Launcher: 3.1.0",
+          "Launcher: 3.1.1",
+          "Launcher: 3.1.4",
+          "1.0.8",
+          "uuid sessionId",
+          "auth_access_token",
+          "keicraft",
+          "keinett",
+          "nodus",
+          "iridium",
+          "mcdonalds",
+          "uranium",
+          "nova",
+          "divinity",
+          "gemini",
+          "mineshafter",
+          "Team-NeO",
+          "DarkLBP",
+          "Launcher X",
+          "PHVL",
+          "Pre-Launcher v6",
+          "LauncherFEnix",
+          "TLauncher",
+          "Titan"
+        ]
+      },
+
+      "removeTriagedMeqs": {
+        "jql": "(cf[12200] != EMPTY OR cf[12201] != EMPTY) AND (comment ~ MEQS_WAI OR comment ~ MEQS_WONTFIX)",
+        "meqsTags": ["MEQS_WAI", "MEQS_WONTFIX"],
+        "removalReason": "Ticket has been triaged."
+      },
+
+      "futureVersion": {
+        "jql": "affectedVersion in unreleasedVersions() AND updated > -1d",
+        "message": "{panel:borderColor=orange}(!) Please do not mark _Unreleased Versions_ as affected. You don't have access to them yet.{panel}\n~{color:#888}-- I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}~"
+      },
+
+      "chk": {
+        "whitelist": ["MC", "MCL", "MCPE"],

```

Comment:
Add MCTEST?

Reply:
MCTEST has no confirmation status and no chk. Do you want to have it enabled anyways?

Reply:
I do, disabling 😅